src/main/java/core/basesyntax/service/CsvFileReaderService.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileReaderService {
+
+    private static List<String> operations = new ArrayList<>();
+
+    public CsvFileReaderService(String filename) {
```

Comment:
Same here, constructor should construct entities. Lets create separate method that will return List<String> operation and accept file name